src/app/store/settings.vuex.ts

```diff
@@ -0,0 +1,36 @@
+import { ActionTree, MutationTree, CommitOptions } from 'vuex';
+import * as Irc from 'irc-framework';
+import {
+  createModule, mutation, action, extractVuexModule,
+} from 'vuex-class-component';
+import Vue from 'vue';
+import { Watch } from 'vue-property-decorator';
+
+const VuexModule = createModule({
+  strict: false,
+});
+export default class SettingsModule extends VuexModule {
+  fontSize: Number = 14;
+
+  indicator: string = ' (!)';
+
+  allChatFeatures = true;
```

Comment:
Maybe this should just exist in the frontend? Imagine a three-state toggle/checkbox - if all are off, set to off. All on, set to on. Some combination, set to the intermediate state. Changing to on sets all on, changing to off sets all off.